app/util/hashToSymbol.js

```diff
@@ -0,0 +1,9 @@
+import { TOKENS } from '../core/constants'
+
+const hashToSymbol = hash =>
+  Object.keys(TOKENS).find(currentSymbol => {
+    const currentHash = TOKENS[currentSymbol].networks['1'].hash
```

Comment:
Ideally we would actually pass the network id into this function as well for testing on other networks in the future (and testnet right now)

Reply:
I think you can do this using lodash get as well. its either get(TOKENS, `${currentSymbol}.networks.1.hash`) or get(TOKENS, [currentSymbol, 'networks', '1', 'hash'], default goes here)